Form/Step.php

```diff
@@ -62,6 +70,26 @@ public static function createFromConfig($number, array $config) {
 	}
 
 	/**
+	 * @param array $options
+	 */
+	public function setOptions(array $options) {
```

Comment:
Either the type hint or the `is_array` check can be removed. Having both is useless. I'm tending slightly into removing the type hint to be consistent with the type handling of the other setters.